site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -875,6 +903,124 @@ This strategy instructions FusionAuth when to append a unique suffix to the user
++
+:advanced_feature!:
++
+:enterprise_feature: WebAuthn roaming authenticators
+include::../shared/_enterprise-edition-blurb-api.adoc[]
++
+:enterprise_feature!:
+
+[field]#tenant.webAuthnConfiguration.reauthenticationWorkflow.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.41.0#::
+Whether or not this tenant has the WebAuthn reauthentication workflow enabled. The reauthentication workflow will automatically prompt a user to authenticate using WebAuthn on repeat logins from the same device.
++
+:advanced_feature: WebAuthn
+include::../shared/_advanced-edition-blurb-api.adoc[]
++
+:advanced_feature!:
+
+[field]#tenant.webAuthnConfiguration.reauthenticationWorkflow.userVerificationRequirement# [type]#[String]# [optional]#Optional# [default]#defaults to `required`# [since]#Available since 1.41.0#::
+Determines the user verification requirement for WebAuthn passkey registration and authentication when using the reauthentication workflow. The possible values are:
++
+ * `discouraged` - If possible, do not require user verification during registration and authentication.
+ * `preferred` - Prefer authenticators that support user verification during registration and authentication but allow others.
+ * `required` - Only authenticators that support user verification will be allowed.
++
+It is _highly_ recommended to use the `required` option for the reauthentication workflow.
++
+:advanced_feature: WebAuthn
+include::../shared/_advanced-edition-blurb-api.adoc[]
++
+:advanced_feature!:
+
+[field]#tenant.webAuthnConfiguration.relyingPartyId# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**# [since]#Available since 1.41.0#::
+The value this tenant will use for the Relying Party Id in WebAuthn ceremonies. Passkeys can only be used to authenticate on sites with the same Relying Party Id they were registered. This value must match your fully qualified domain name or be a registrable suffix of your fully qualified domain. For example, if your domain is `auth.example.com`, you could use `auth.example.com` or `example.com` but not `m.auth.example.com` or `com`.
++
+When this parameter is omitted, FusionAuth will use `null` for the Relying Party Id in passkey creation and request options. A `null` value in the WebAuthn client API will use the current fully qualified domain.
++
+:advanced_feature: WebAuthn
+include::../shared/_advanced-edition-blurb-api.adoc[]
++
+:advanced_feature!:
+
+[field]#tenant.webAuthnConfiguration.relyingPartyName# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**# [since]#Available since 1.41.0#::
+The value this tenant will use for the Relying Party name in WebAuthn ceremonies. This value is used only for display and may be shown by browser or OS dialogs during WebAuthn ceremonies.
++
+When this parameter is omitted, FusionAuth will use the [field]#tenant.issuer# value in passkey creation options.
```

Comment:
```suggestion
When this parameter is omitted, FusionAuth will use the [field]#tenant.issuer# value.
```

Unless there's some reason for the 'passkey creation options' text?

Reply:
Just to provide more context about how the value is used because it may not be obvious what that value is for. Maybe there's a better place to go into detail about that. I assume we'll have a developer guide or something like that for WebAuthn.